sklearn/metrics/_plot/roc_curve.py

```diff
@@ -36,6 +36,21 @@ class RocCurveDisplay:
 
     figure_ : matplotlib Figure
         Figure containing the curve.
+
+    Examples
+    --------
+    >>> import pytest
+    >>> plt = pytest.importorskip('matplotlib.pyplot')
+    >>> import numpy as np
+    >>> from sklearn import metrics
+    >>> y = np.array([0, 0, 1, 1])
+    >>> pred = np.array([0.1, 0.4, 0.35, 0.8])
+    >>> fpr, tpr, thresholds = metrics.roc_curve(y, pred)
+    >>> roc_auc = metrics.auc(fpr, tpr)
+    >>> metrics.RocCurveDisplay(fpr=fpr, tpr=tpr, roc_auc=roc_auc,\
+                                estimator_name='example estimator').plot()
```

Comment:
We can create the `Display` object and then call plot on another line. The plot to call will be skipped.